src/rollup/BOLDUpgradeAction.sol

```diff
@@ -433,17 +442,6 @@ contract BOLDUpgradeAction {
             PROXY_ADMIN_SEQUENCER_INBOX.upgrade(sequencerInbox, IMPL_SEQUENCER_INBOX);
         }
 
-        // verify
```

Comment:
Is there manual checklist somewhere that we could add these to?

Reply:
not yet but we can make something